### PR TITLE
docs: Update release notes for tracker

### PR DESCRIPTION
**Pull request by enricocolasante**

### CODE REVIEW

File: releases/2.40/README.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+## Update Dashboard
+- From 2.40 when updating a `Dashboard`, user is required to have `METADATA_WRITE` permission to the `Dashboard` itself and `METADATA_READ` permission to all referenced objects in all `DashboardItems`. The Administrator can use function [Cascade sharing](https://docs.dhis2.org/en/use/user-guides/dhis-core-version-master/analysing-data/dashboards.html#cascade-sharing-of-visualizations-on-the-dashboard) in order to grant required sharing permissions when needed.
 - From 2.40 there is new column in Analytics tables(ounamehierarchy). New export of analytics tables is required.
+
+## API
+
+* **Old tracker APIs deprecated:** In previous release old tracker API endpoints were deprecated. (see [deprecated-features](https://github.com/dhis2/dhis2-releases/blob/master/releases/deprecated-features.md) ). We encourage developers of apps and extensions do transition over to move away from old endpoints. Deprecated API endpoints are:
```
Comment (Bekkalizer) [inline releases/2.40/README.md:9]:
```suggestion
* **Old tracker APIs deprecated:** In the previous release, the old tracker API endpoints were deprecated. (see [deprecated-features](https://github.com/dhis2/dhis2-releases/blob/master/releases/deprecated-features.md) ). We encourage developers of apps and extensions transition and move away from the old endpoints. The deprecated API endpoints are:
```

File: releases/2.40/README.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+   - /api/events
+   - /api/relationships
+   ```
+   This endpoint should be used instead:
+   ```
+   - /api/tracker/
+   ```
+   [Docs tracker endpoint](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-240/tracker.html) | [Docs old tracker endpoints](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-240/old-tracker.html)
+
+* **Removed status field in bundleReport:** Import summaries have the following overall structure [defined here](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-240/tracker.html#import-summary-structure). Status field in `bundleReport` property was redundant, hence it was removed. `Status` field in the root of the import summary should be used.
```
Comment (Bekkalizer) [inline releases/2.40/README.md:22]:
```suggestion
* **Removed the status field in bundleReport:** Import summaries have the following overall structure [defined here](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-240/tracker.html#import-summary-structure). The status field in `bundleReport` property was redundant and hence it was removed. The `Status` field in the root of the import summary should be used instead.
```

File: releases/deprecated-features.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ The following features are deprecated. You can still use them now, but the DHIS2
 
 |Feature|Deprecation first announced|Planned end of support<br>(version)|
 |:---------|:---:|:---:|
+|Old tracker endpoints are deprecated, and replaced by `/api/tracker` endpoint |Sep 2022|2.41.0|
```
Comment (Bekkalizer) [inline releases/deprecated-features.md:15]:
```suggestion
|The old tracker endpoints are deprecated and replaced by `/api/tracker` endpoint |Sep 2022|2.41.0|
```

**Review by Bekkalizer — CHANGES_REQUESTED** [review]
Looks good! Made some minor language suggestions.

File: releases/2.40/README.md (hunk 1 above)
Comment (teleivo) [inline releases/2.40/README.md:9]: > **Old tracker APIs deprecated:** In previous release old tracker API endpoints were deprecated.

Did we miss adding this note in the last release? Or is it just a reminder?

I would replace `In previous release` with the exact version they were deprecated 😄

File: releases/2.40/README.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
 - From 2.40 there is new column in Analytics tables(ounamehierarchy). New export of analytics tables is required.
+
+## API
+
+* **Old tracker APIs deprecated:** In previous release old tracker API endpoints were deprecated. (see [deprecated-features](https://github.com/dhis2/dhis2-releases/blob/master/releases/deprecated-features.md) ). We encourage developers of apps and extensions do transition over to move away from old endpoints. Deprecated API endpoints are:
+   ```
+   - /api/trackedEntityInstances
+   - /api/enrollments
+   - /api/events
+   - /api/relationships
```
Comment (teleivo) [inline releases/2.40/README.md:14]: Are these all the old tracker endpoints? I wonder if there are some that exist in old tracker but were just not implemented in new tracker 🤔

File: releases/2.40/README.md [hunk 5]
```diff
@@ -1,5 +1,22 @@
...
+   ```
+   This endpoint should be used instead:
```
Comment (teleivo) [inline releases/2.40/README.md:16]: Use [tracker endpoints](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-240/tracker.html) instead.
```suggestion
   This endpoint should be used instead:
```

as you list endpoint**s** but then say use endpoint (singular) instead.

**Review by teleivo — APPROVED** [review]

**Pull request merged**